tools/actions/commit-queue.sh

```diff
@@ -40,8 +40,8 @@ for pr in "$@"; do
   fi
 
   # Skip PR if CI is still running
-  if ncu-ci url "https://github.com/${OWNER}/${REPOSITORY}/pull/${pr}" 2>&1 | grep "^Result *PENDING"; then
-    echo "pr ${pr} skipped, CI still running"
+  if gh pr checks "$pr" | grep -E "pending|fail"; then
```

Comment:
I don't think we want to skip if the CI has failed – imo it's useful to have the CQ comment to the PR explaining why it can't land, rather than silently ignoring the error.
We can use the `-q` flag of grep, add `\t` to limit the risk of false positives:
```suggestion
  if gh pr checks "$pr" | grep -q "\tpending\t"; then
```